socorro/tests/external/es/test_crashstorage.py

```diff
@@ -336,245 +328,6 @@ def test_fatal_failure(self, espy_mock):
                 crash_id=crash_id,
             )
 
-    @mock.patch("elasticsearch.client")
```

Comment:
These tests mock everything out and hard-code messages that Elasticsearch 1.4 returns. What happens when we update to a later version of Elasticsearch? Does this help us? _shrug_

Further, when we hit problems with indexing, we really want to write code to fix things so the problems _don't_ occur. 

Having these tests doesn't help us.